x-pack/plugins/observability_solution/observability_onboarding/public/application/onboarding_flow_form/use_custom_cards_for_category.ts

```diff
@@ -57,8 +57,25 @@ export function useCustomCardsForCategory(
           integration: '',
         },
         {
-          id: 'synthetics-generated',
-          type: 'generated',
+          id: 'otel-virtual',
+          type: 'virtual',
+          title: 'OpenTelemetry',
+          description: 'Collect distributed traces with OpenTelemetry',
+          name: 'otel',
+          categories: ['observability'],
+          icons: [
+            {
+              type: 'svg',
+              src: '/XXXXXXXXXXXX/plugins/home/assets/logos/opentelemetry.svg',
```

Comment:
out of curiosity how the `/XXXXXXXXXXXX/` is handled?

Reply:
We are using the integrations API (all of the display stuff is routed through Fleet UI components), and this seems to be the path convention it uses (all non-EUI icon `src` look like this on the client in the packages list). I didn't trace this to where it gets resolved, but some part of the call stack is in [this hook](https://github.com/elastic/kibana/blob/0b9847dbcf9cf1091800c694cea632a73dd31b1d/x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/home/hooks/use_available_packages.tsx#L97). When the integrations are fetched, they follow one of two conventions:

```typescript
{
  type: 'svg',
  src: `/XXXXXXXXXXXX/plugins/home/...`
}
{
  type: 'eui',
  src: 'apmApp'
}
```

That being said, I did a little searching around while researching this response and it looks like you can instead do:

```typescript
core.http.basePath.prepend(`/plugins/${PLUGIN_NAME}/assets/${ICON_NAME}.svg`)
```

I'm going to try this as it looks much cleaner. And we can probably keep the otel icon in the onboarding plugin instead of `home`.

Reply:
thanks for the explanation.  I agree, much cleaner 🥳